components/tabs/index.css

```diff
@@ -165,81 +96,97 @@
 
 	/* Friends should align to the top of the tabs */
 	vertical-align: top;
-
-	background: linear-gradient(to var(--mod-tabs-list-background-direction, var(--spectrum-tabs-list-background-direction)), var(--highcontrast-tabs-divider-background-color, var(--mod-tabs-divider-background-color, var(--spectrum-tabs-divider-background-color))) 0 var(--mod-tabs-divider-size, var(--spectrum-tabs-divider-size)), transparent var(--mod-tabs-divider-size, var(--spectrum-tabs-divider-size)));
```

Comment:
I'm not clear on what this linear gradient's purpose was initially, but since we don't have a divider anymore, I pulled it out.